rest-server/src/routes/job.js

```diff
@@ -35,7 +35,7 @@ router.route('/:jobName')
     .get(jobController.get)
 
     /** PUT /api/v1/jobs/:jobName - Update job */
-    .put(tokenConfig.check, param.validate(jobConfig.schema), jobController.update)
+    .put(tokenConfig.check, param.validate(jobConfig.schema, {'params': ['jobName']}), jobController.update)
```

Comment:
As we discussed yesterday, please use a separate middleware to synthesis req.body.